instructions.md

```diff
@@ -0,0 +1,89 @@
+# Installing LGTM Enterprise on Google Cloud
+## Preparation
+Throughout this setup guide a few variables are used that are useful to set in advance. Set the Google Cloud project you want to deploy to, the zone to create any zonal resources in and a unique name for this deployment. If you are upgrading an existing installation, set these values to the same values used when LGTM was first set up.
+
+```console
+project="my-project"
+deployment="lgtm"
+zone="us-central1-a"
```

Comment:
Swap to match the same order as referred to in the details above (not necessary, I know, I just think it reads nicer to keep things in the same order):
```suggestion
zone="us-central1-a"
deployment="lgtm"
```